functions/Get-DbaTable.ps1

```diff
@@ -0,0 +1,176 @@
+﻿function Get-DbaTable
+{
+<#
+<#
+.SYNOPSIS
+Returns a summary of information on the table
+.DESCRIPTION
+Shows table information around table row and data sizes and if it has any table type information. 
+.PARAMETER SqlInstance
+SQLServer name or SMO object representing the SQL Server to connect to. This can be a
+collection and recieve pipeline input
+.PARAMETER SqlCredential
+PSCredential object to connect as. If not specified, currend Windows login will be used.
+.PARAMETER Database
+Define the database you wish to search
+.PARAMETER IncludeSystemDBs
+Switch parameter that when used will display system database information
+.PARAMETER Table
+Define a specific table you would like to query
+.PARAMETER Silent 
+Use this switch to disable any kind of verbose messages
+	
+.NOTES 
+Original Author: Stephen Bennett, https://sqlnotesfromtheunderground.wordpress.com/
+dbatools PowerShell module (https://dbatools.io, clemaire@gmail.com)
+Copyright (C) 2016 Chrissy LeMaire
+This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+You should have received a copy of the GNU General Public License along with this program.  If not, see <http://www.gnu.org/licenses/>.
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.	
+	
+.LINK
+https://dbatools.io/Get-DbaTable
+.EXAMPLE
+Get-DbaTable -SqlInstance DEV01 -Database Test1
+Return all tables in the Test1 database
+.EXAMPLE
+Get-DbaTable -SqlInstance DEV01 -Database MyDB -Table MyTable
+Return only information on the table MyTable from the database MyDB
+
+#>
+	[CmdletBinding()]
+	param ([parameter(ValueFromPipeline, Mandatory = $true)]
+		[Alias("ServerInstance", "SqlServer")]
+		[object[]]$SqlInstance,
+		[System.Management.Automation.PSCredential]$SqlCredential,
+		[switch]$IncludeSystemDBs,
+		[switch]$Silent,
```

Comment:
Please make silent the last parameter in the function. Most users will never use it and the order the parameters are defined in is also the one the user tabs through the possible parameters